api/src/test/java/marquez/db/OpenLineageDaoTest.java

```diff
@@ -94,6 +95,41 @@ void testUpdateMarquezModelLifecycleStateChangeFacet() {
         .isEqualTo("TRUNCATE");
   }
 
+  @Test
+  void testUpdateMarquezModelDatasetWithColumnLineageFacet() {
+    Dataset dataset =
+        new Dataset(
+            LineageTestUtils.NAMESPACE,
+            DATASET_NAME,
+            LineageEvent.DatasetFacets.builder()
+                .columnLineage(
+                    new LineageEvent.ColumnLineageFacet(
+                        PRODUCER_URL,
+                        SCHEMA_URL,
+                        Collections.singletonList(
+                            new LineageEvent.ColumnLineageOutputColumn(
+                                "output_column",
+                                Collections.singletonList(
+                                    new LineageEvent.ColumnLineageInputField(
+                                        "namespace", "datasetname", "fieldname")),
+                                "transformation_description",
+                                "transformation_type"))))
+                .build());
+
+    JobFacet jobFacet = new JobFacet(null, null, null, LineageTestUtils.EMPTY_MAP);
+    UpdateLineageRow writeJob =
+        LineageTestUtils.createLineageRow(
+            dao, WRITE_JOB_NAME, "COMPLETE", jobFacet, Arrays.asList(), Arrays.asList(dataset));
+
+    assertThat(
+            writeJob.getOutputs().get().stream().toList().stream()
+                .findAny()
+                .orElseThrow()
+                .getColumnLineageRows())
```

Comment:
Great, this is how the test should look like. 
I think it's worth testing column lineage row columns either.

Reply:
Done.